core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/PathPropUtils.kt

```diff
@@ -25,14 +26,15 @@ fun makePathProps(
     blockIdx: BlockId,
     beginOffset: Offset<Block> = Offset(0.meters),
     endOffset: Offset<Block> = blockInfra.getBlockLength(blockIdx),
-    routes: List<String>? = null
+    routes: List<String>? = null,
+    temporarySpeedLimitManager: TemporarySpeedLimitManager? = null,
```

Comment:
```suggestion
```

Not needed anymore

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/9837/commits/821eabe352cb73cebddb301a145ae69b6bcf3383